docs/how-to/use-padlocal-protocol.md

```diff
@@ -51,11 +51,11 @@ curl https://api.chatie.io/v0/hosties/$WECHATY_TOKEN (个人随机token)
 ```shell
 #1  默认配置
 export WECHATY_PUPPET="wechaty-puppet-service"
-export WECHATY_PUPPET_SERVICE_TOKEN="puppet_padlocal_XXXXXX"
+export WECHATY_PUPPET_SERVICE_TOKEN="1fe5f846-3cfb-401d-b20c-XXXXX"
```

Comment:
Why do you remove the prefix of token, which should be: `puppet-padlocal`?

Reply:
![image](https://user-images.githubusercontent.com/22414051/166180876-4556d70a-ecff-4402-9035-ff4a77f4cedf.png)
![image](https://user-images.githubusercontent.com/22414051/166180912-d8278589-55a4-41bf-ae64-575b8c399b38.png)

![image](https://user-images.githubusercontent.com/22414051/166180953-ed72efb3-da59-481b-9e9d-09e8af64a3e3.png)
![image](https://user-images.githubusercontent.com/22414051/166180975-5384c6fc-cd15-4039-bf53-98cfcb9d869f.png)

![image](https://user-images.githubusercontent.com/22414051/166181032-06150290-b8ba-4501-8fd0-cd8dd2499adb.png)

**It seems that _WECHATY_PUPPET_SERVICE_TOKEN_ should be _WECHATY_TOKEN(1fe5f846-3cfb-401d-b20c-XXXXX)_ instead of _WECHATY_PUPPET_PADLOCAL_TOKEN(puppet_padlocal_XXXXXX)_**

Reply:
> Why do you remove the prefix of token, which should be: `puppet-padlocal`?

大佬，帮忙审核下。